src/opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp

```diff
@@ -1486,6 +1486,13 @@ namespace {
                 auto gas_target = record.getItem("GAS_TARGET").get<UDAValue>(0);
                 auto water_target = record.getItem("WATER_TARGET").get<UDAValue>(0);
                 auto liquid_target = record.getItem("LIQUID_TARGET").get<UDAValue>(0);
+                double guide_rate = -1;
+                std::string guide_rate_def;
```

Comment:
This should not be internalized as a string - should rather create an enum:
```
class enum GuideRateDef {
      OIL = 1,
      GAS = 2,
      WAT = 3, 
      ....
}
```

Reply:
Reuse GuideRate::GuideRatePhaseEnum ?

Reply:
If the possible values agree with that enum - yes that would be good - otherwise create a new one.